bot/cogs/cyber.py

```diff
@@ -211,63 +212,40 @@ def __init__(self, bot: Bot):
 
     @command()
     async def game(self, ctx: Context):
```

Comment:
Add a docstring explaining the command. The `:help` function uses this docstring to generate a description, see my screenshot above.